multinet/__init__.py

```diff
@@ -27,6 +27,9 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     app.register_blueprint(uploaders.nested_json.bp, url_prefix="/api/nested_json")
     app.register_blueprint(uploaders.d3_json.bp, url_prefix="/api/d3_json")
 
+    app.register_blueprint(downloaders.csv.bp, url_prefix="/api/csv")
```

Comment:
Agreed that this should probably be

```suggestion
    app.register_blueprint(downloaders.csv.bp, url_prefix="/api/csv/download")
```

but also see the discussion below about where this section of API should ultimately go.

In general, I don't think it's right to set a full-on blueprint at the single route where it wants to be called. The point of setting this prefix is for *multiple* endpoints to be sequestered together under a common prefix, rather than giving the actual route to a single endpoint.